docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,56 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Quicker path to troubleshooting in Kubernetes Monitoring
+
+<!-- Beverly Buchanan --> 
+
+_Generally available in Grafana Cloud_
+
+You can use Kubernetes Monitoring to more quickly identify and troubleshoot issues with these views:
+
+- The home page shows a [snapshot of issues](/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/navigate-k8s-monitoring/#see-the-issues-at-a-glance) that reach a specific threshold. 
+  
+  (ADD SCREENSHOT AFTER CROW's NEST LANGUAGE REMOVED)
+
+  This view shows:
+
+  - Pods that have been in a non-running state for more than 15 minutes
+  - Nodes with CPU usage above 90% for more than five minutes
+  - Nodes using more than 90% of memory for more than five minutes
+  - Persistent Volumes with capacity above 90%
+  - Node disks with capacity above 90%
+
+- The [**Cluster Navigation** page](docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/navigate-k8s-monitoring/#drill-down-into-data/) provides tabs that allow you to quickly switch between the Cluster, namespace, workload, and Node views.
+
+  (ADD SCREENSHOT)
+
+## Configuration additions in Kubernetes Monitoring
+
+<!-- Beverly Buchanan --> 
+<!-- Cloud --> 
+
+_Generally available in Grafana Cloud_
+
+You have these options when configuring Kubernetes Monitoring:
+
+- Grafana Kubernetes Monitoring Helm chart: To send infrastructure data to Grafana Cloud, you can configure easily using the Helm chart. With this method, you can select toggle buttons to gather metrics, logs, events, traces, and cost metrics.
```

Comment:
"configure easily" without a thing you're configuring as part of the phrase sounds odd. I don't have a suggestion because I'm not sure what the missing piece is. Also if we can avoid the repetition of "you can" in both sentences, that would be better.

Reply:
Rewrote a lot of content. Good point.